kernel/arch/dreamcast/hardware/scif.c

```diff
@@ -193,23 +193,20 @@ int scif_init_fake() {
     return 0;
 }
 
-/* Initialize the SCIF port; baud_rate must be at least 9600 and
-   no more than 57600. 115200 does NOT work for most PCs. */
+/* Initialize the SCIF port; */
 /* recv trigger to 1 byte */
 int scif_init() {
-    int i;
```

Comment:
Why did this variable get removed? There's no reason to change this to defining the variable `i` in both of the for loops below, and it does not match with the style of the rest of the code in KOS.

Reply:
Sorry for that, I'll get back it.
I played with the values of the cycles and apparently played too much ))